templates/download/iot/raspberry-pi-2-3.html

```diff
@@ -70,14 +72,47 @@ <h3 class="p-list-step__title">
             </ol>
           </div>
         </li>
-        {% include "./_first-boot-setup.html" with number=5 %}
-        {% include "./_login.html" with number=6 %}
+        {% include "./_login.html" with number=4 %}
```

Comment:
This login step doesn't apply. 

Copy to use: "When prompted to log in, use "ubuntu" for the username and the password. You will be asked to change this default password after you log in."